samples/nrf9160/gnss/README.rst

```diff
@@ -20,7 +20,7 @@ The sample supports the following development kit:
 
 .. table-from-sample-yaml::
 
-.. include:: /includes/spm.txt
+.. include:: /includes/tfm.txt
```

Comment:
The GNSS sample still uses SPM: https://github.com/nrfconnect/sdk-nrf/blob/261b4e2ee6a2c8c74d77430d32ed76356fbe0784/samples/nrf9160/gnss/prj.conf#L51

Reply:
True, GNSS links with a library that is only available in FP_HARDABI, which is currently not supported by TF-M.

Reply:
reverted to spm.txt